protocol/communities/manager.go

```diff
@@ -607,13 +607,13 @@ func (m *Manager) CreateCommunityTokenPermission(request *requests.CreateCommuni
 		return nil, nil, err
 	}
 
-	if community.IsOwner() {
+	if community.IsControlNode() {
```

Comment:
For all of these where we publish a community description, I think we still want to just check for `isOwner()` no?

Reply:
fixed